packages/client/tests/functional/queryRaw-send-type-hints/prisma/_schema.ts

```diff
@@ -1,18 +1,20 @@
-export default ({ provider, previewFeatures }) => {
+import testMatrix from '../_matrix'
+
+export default testMatrix.setupSchema(({ provider, previewFeatures }) => {
   return /* Prisma */ `
-  generator client {
-    provider = "prisma-client-js"
-    previewFeatures = [${previewFeatures}]
-  }
-  
-  datasource db {
-    provider = "${provider}"
-    url      = env("DATABASE_URI_${provider}")
-  }
-  
-  model Entry {
-    id     Int @id @default(autoincrement())
-    binary Bytes
-  }
+    generator client {
+      provider = "prisma-client-js"
+      previewFeatures = [${previewFeatures}]
+    }
+    
+    datasource db {
+      provider = "${provider}"
+      url      = env("DATABASE_URI_${provider}")
+    }
+    
+    model Entry {
+      id     Int @id @default(autoincrement())
```

Comment:
Can this benefit from `idForProvider`?

Reply:
sure!

Reply:
https://github.com/prisma/prisma/pull/13463#discussion_r883677786